docs/Server-Setup.md

```diff
@@ -83,6 +83,17 @@ Start the CodaLab service as follows:
 
     ./codalab_service.py start -bd
 
+If your default python version is not python 3.6+, then you might have to use:
+
+    python3 codalab_service.py start -bd
```

Comment:
Don't need this since `python3.6` is specified in `codalab_service.py` already.

Reply:
I still receive the following error if I just run ./codalab_service.py
`bash: ./codalab_service.py: /usr/bin/python3: bad interpreter: No such file or directory`

Reply:
replace the first line with `#!/usr/bin/env python3`

Reply:
Is it because I used anaconda instead of pip to install my python3?

Reply:
Yes I originally changed the line to `#!/usr/bin/env python3` but then I don't know if I should update the master branch with this change. I think `#!/usr/bin/env python3` is preferred in shebang as it doesn't assume that it's running on a Ubuntu system. Any reason why we chose to use `#!/usr/bin/python3`?

Reply:
It should be `#!/usr/bin/env python3`

Reply:
Ok I will create a separate PR and fix that!